wagtail/images/wagtail_hooks.py

```diff
@@ -116,7 +116,10 @@ def register_image_operations():
         ('width', image_operations.WidthHeightOperation),
         ('height', image_operations.WidthHeightOperation),
         ('scale', image_operations.ScaleOperation),
-        ('jpegquality', image_operations.JPEGQualityOperation),
+        # For backward compatibility
+        ('jpegquality', image_operations.QualityOperation),
+        ('quality', image_operations.QualityOperation),
+        ('webplossless', image_operations.WebPLossless),
```

Comment:
I feel like this could be a nice extension to the `format` filter: `format-webp-lossless`